src/main/java/com/redhat/rhjmc/containerjfr/platform/internal/KubeApiPlatformClient.java

```diff
@@ -80,13 +80,7 @@
                     .map(V1Service::getSpec)
                     .peek(spec -> logger.trace("Service spec: " + spec.toString()))
                     .filter(s -> s.getPorts() != null)
-                    .flatMap(
-                            s ->
-                                    s.getPorts().stream()
-                                            .map(
-                                                    p ->
-                                                            new ServiceRef(
-                                                                    s.getClusterIP(), p.getPort())))
+                    .flatMap(s -> s.getPorts().stream().map(p -> new ServiceRef(s.getClusterIP())))
```

Comment:
The cluster IP is definitely not a service URL

Reply:
Should be fixed.